clients/mobile/src/utility/validator.ts

```diff
@@ -0,0 +1,80 @@
+interface PasswordPolicy {
+  minimum_length: number;
+  require_lowercase: boolean;
+  require_numbers: boolean;
+  require_uppercase: boolean;
+  require_symbols: boolean;
+}
+
+const passwordPolicy: PasswordPolicy = {
+  minimum_length: 10,
+  require_lowercase: true,
+  require_numbers: true,
+  require_uppercase: false,
+  require_symbols: true
+}
+
+export const validateAndReturnRequiredFieldError = (value: string): string => {
+  return value.trim().length === 0 ? 'This field is required' : ''
+}
+
+export const validateEmailAndGetErrorMessage = (value: string): string => {
+  const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
+  return !emailRegex.test(value) ? 'Invalid email address' : ''
+}
+
+export const validatePhoneNumberAndGetErrorMessage = (value: string): string => {
+  const phoneRegex = /^\+?[1-9]\d{1,14}$/
+  return !phoneRegex.test(value) ? 'Invalid phone number' : ''
+}
+
+export const validatePasswordAndGetErrorMessage = (value: string): string => {
```

Comment:
maybe this should be called checkErrorsInPassword and it should return null if there are no errors

Reply:
Update the name to `checkErrorsInPassword ` and return null if the error is not found.[Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/138/commits/0e9819bab559227cbd4d26afdb38b6451c5b83ff)